deployments/goerli/weth/configuration.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Compound WETH",
+  "symbol": "cWETHv3",
+  "baseToken": "WETH",
+  "baseTokenAddress": "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
+  "borrowMin": "0.1e18",
+  "governor": "0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399",
+  "pauseGuardian": "0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000e18",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
```

Comment:
TODO: Need to update these rates to the WETH model (which is still under discussion I think)

Reply:
Done. Updated to the params provided by Gauntlet [here](https://www.comp.xyz/t/initialize-compound-iii-weth-on-ethereum/3737/21). I had to annualize the per-second params they provided.

Reply:
Of course it's JSON so we can't comment on the conversions or write them as expressions 😭